Src/CSharpier.Tests/CommandLineFormatterTests.cs

```diff
@@ -213,10 +216,11 @@ public void Format_Checks_Formatted_File()
     [TestCase("TestFile.g.i.cs")]
     public void Format_Skips_Generated_Files(string fileName)
     {
+        var context = new TestContext();
         var unformattedFilePath = fileName;
-        WhenAFileExists(unformattedFilePath, UnformattedClassContent);
+        context.WhenAFileExists(unformattedFilePath, UnformattedClassContent);
 
-        var result = this.Format();
+        var result = context.Format();
```

Comment:
It feels weird to have a the `Format()` on a `context`. Like, it's not just a context anymore.
Maybe you can call it `CommandLineFormatTester`?

Or maybe you can have the `Format()` method be outside that class, and pass the context in as a parameter?

Reply:
I like passing context to a format method. It solves some of the problems with the other potential solutions.